### PR TITLE
Add type=system sample

### LINKED ISSUE #617: Sample for <amp-social-share type="system">
@ithinkihaveacat would be great if you could update the amp-social-share sample.

### PR DESCRIPTION
**Pull request by ithinkihaveacat**
Fixes #617.

Test via <https://cdn.ampproject.org/c/s/xbyexample.appspot.com/components/amp-social-share/>.

### CODE REVIEW

File: src/20_Components/amp-social-share.html [hunk 1]
```diff
@@ -90,5 +95,27 @@
...
+    * The user must be viewing the AMP using Chrome on Android. (The only supported browser as of February 2017.)
+    * An appropriate token (origin-specific) must be embedded on the site to activate the feature. ([Apply for a token](https://goo.gl/GR5YLI); [more information about the process](https://developers.google.com/web/updates/2016/10/navigator-share).)
+    
+  For this example, we've embedded the tokens via `<meta>` elements [in the document `<head>`](#setup).
+  -->
+
+  <amp-social-share type="system"></amp-social-share>
+
+  <!--
+  This type supports one option: if `data-mode="replace"` is specified, then all
```
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: It's not possible to demo this... Would screenshots be useful?

File: src/20_Components/amp-social-share.html [hunk 2]
```diff
@@ -18,6 +18,11 @@
     Ensure you've configured your canonical URL.
   -->
   <link rel="canonical" href="<%host%>/components/amp-social-share/">
+  <!--
+    Origin trial tokens for `ampbyexample.com` and `*.cdn.ampproject.org`. (Needed for `type=system`; [see below](#native-share-dialog-some-browsers-).)
+  -->
+  <meta http-equiv="origin-trial" data-feature="Web Share" data-expires="2017-03-06" content="AtEHR9Ke52xpgqsGbRi+U1kDOYSDQTfzeITL+Q/nvBiOx/qCW/vMWQxyt1y8+6688/9pjfdnCc0sJBqgkwQkjwsAAABLeyJvcmlnaW4iOiJodHRwczovL2JlZWJvLnJlZDo0NDMiLCJmZWF0dXJlIjoiV2ViU2hhcmUiLCJleHBpcnkiOjE0ODg4MTA2NjR9">
```
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Waiting on a token for `ampbyexample.com`; should be ready in a day or two. Unfortunately they only last a couple of months.

File: src/20_Components/amp-social-share.html (hunk 1 above)
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: An animated GIF would be wonderful...

File: src/20_Components/amp-social-share.html (hunk 2 above)
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: Do you get notified once it becomes invalid?

Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Yes I get an email and then I apply for another one. It's not ideal...

File: src/20_Components/amp-social-share.html [hunk 3]
```diff
@@ -90,5 +95,37 @@
...
+  This type supports one option: if `data-mode="replace"` is specified, then all
+  other share icons will be suppressed. (That is, the only share icon the user
+  will see is the system share icon.)
+
+  Since it's not possible to demo this feature within the context of this
+  example, here's some screenshots to illustrate the difference.
+
+  `<amp-social-share type="system"></amp-social-share>`:
+  -->
+  <amp-img src="/img/system-share1.png" height="576" width="324"></amp-img>
```
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Is there some way to not display the actual code in the "middle" column?

File: src/20_Components/amp-social-share.html (hunk 1 above)
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: What would the animated GIF show? I can't think of a suitable transition. (Have added two separate screenshots for now.)

Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: I think the GIF could show the transition from clicking the button to the native share dialog.

File: src/20_Components/amp-social-share.html (hunk 3 above)
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: In this case, it's best to simply declare the image inside the comment -> this will render the screenshot inside the docs.

File: src/20_Components/amp-social-share.html (hunk 1 above)
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Added in 467dcdb8b4b98f1bd4c57ba81acb753052992d0d.

File: src/20_Components/amp-social-share.html (hunk 2 above)
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: I applied for an `ampbyexample.com` token using the separate email address you provided--did it arrive?

File: src/20_Components/amp-social-share.html (hunk 3 above)
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Moved in 467dcdb8b4b98f1bd4c57ba81acb753052992d0d.

File: src/20_Components/amp-social-share.html [hunk 4]
```diff
@@ -90,5 +95,41 @@
 
   <amp-social-share type="pinterest" data-param-media="https://ampbyexample.com/img/amp.jpg"></amp-social-share>
 
+  <!-- #### Native Share Dialog (some browsers) -->
```
Comment (kul3r4) [inline src/20_Components/amp-social-share.html]: can we say "Chrome on Android" instead of "some browsers"?

File: src/20_Components/amp-social-share.html (hunk 2 above)
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: yes - thanks!

Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Added in 1d2937f5b3a87fa9bf97ee88456c6692306dc199.

File: src/20_Components/amp-social-share.html (hunk 4 above)
Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Changed in 4919e225b6aba1a3bff981a5155115033f83a66c.

File: src/20_Components/amp-social-share.html [hunk 5]
```diff
@@ -90,5 +95,41 @@
...
+  -->
+
+  <amp-social-share type="system"></amp-social-share>
+
+  <!--
+  This type supports one option: if `data-mode="replace"` is specified, then all
+  other share icons will be suppressed. (That is, the only share icon the user
+  will see is the system share icon.)
+
+  Since it's not possible to demo this feature within the context of this
```
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: It actually is possible to demo the feature in this sample -> simply open the sample on an android device. Maybe rephrase to:

>Open this sample in Chrome on Android to see it in action. Here are also some screenshots to illustrate the difference.

Comment (ithinkihaveacat) [inline src/20_Components/amp-social-share.html]: Oh, what I was trying to convey is that it's not possible to include the string `<amp-social-share type="system" data-mode="replace"></amp-img>` anywhere in the source of the sample, because when opened on an Android device, this will cause all other social share buttons to disappear, including those in the "Basic Usage" section, etc.

Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: aaah - my fault
